clusterman/aws/aws_resource_group.py

```diff
@@ -194,6 +194,7 @@ def load(cls, cluster: str, pool: str, config: Any, **kwargs: Any) -> Mapping[st
                     rg = cls(rg_id, **kwargs)
                     matching_resource_groups[rg_id] = rg
             except KeyError:
+                logger.exception(f'Could not load resource group {rg_id}; skipping...')
```

Comment:
For greater clarity, I would also print the exception, just in case the key we don't find isn't the one we expect.

Reply:
`logger.exception` prints the whole traceback, including the exception.